webnn/resources/test_data/constant.json

```diff
@@ -0,0 +1,708 @@
+          12.2890625,
+          13.046875,
+          13.796875,
+          14.5546875,
+          15.3046875,
+          16.0625,
+          16.8125,
+          17.5625
+        ],
+        "type": "float16"
+      }
+    },
+    {
+      "name": "constant float32 4D tensor of int32 type",
+      "inputs": {
+        "start": {
+          "data": 0.22992068529129028,
+          "type": "float32"
+        },
+        "step": {
+          "data": 0.7537541389465332,
+          "type": "float32"
+        }
+      },
+      "outputShape": [2, 2, 2, 3],
+      "type": "int32",
+      "expected": {
+        "name": "output",
+        "shape": [2, 2, 2, 3],
+        "data": [
+          0,
+          0,
+          1,
+          2,
+          3,
+          3,
+          4,
+          5,
+          6,
+          7,
+          7,
+          8,
+          9,
+          10,
+          10,
+          11,
+          12,
+          13,
+          13,
+          14,
+          15,
+          16,
+          16,
+          17
+        ],
+        "type": "int32"
+      }
+    },
+    {
+      "name": "constant float32 4D tensor of uint32 type",
+      "inputs": {
+        "start": {
+          "data": 0.22992068529129028,
+          "type": "float32"
+        },
+        "step": {
+          "data": 0.7537541389465332,
+          "type": "float32"
+        }
+      },
+      "outputShape": [2, 2, 2, 3],
+      "type": "uint32",
+      "expected": {
+        "name": "output",
+        "shape": [2, 2, 2, 3],
+        "data": [
+          0,
+          0,
+          1,
+          2,
+          3,
+          3,
+          4,
+          5,
+          6,
+          7,
+          7,
+          8,
+          9,
+          10,
+          10,
+          11,
+          12,
+          13,
+          13,
+          14,
+          15,
+          16,
+          16,
+          17
+        ],
+        "type": "uint32"
+      }
+    },
+    {
+      "name": "constant float32 4D tensor of int64 type",
+      "inputs": {
+        "start": {
+          "data": 0.22992068529129028,
+          "type": "float32"
+        },
+        "step": {
+          "data": 0.7537541389465332,
+          "type": "float32"
+        }
+      },
+      "outputShape": [2, 2, 2, 3],
+      "type": "int64",
+      "expected": {
+        "name": "output",
+        "shape": [2, 2, 2, 3],
+        "data": [
+          "0",
+          "0",
+          "1",
+          "2",
+          "3",
+          "3",
+          "4",
+          "5",
+          "6",
+          "7",
+          "7",
+          "8",
+          "9",
+          "10",
+          "10",
+          "11",
+          "12",
+          "13",
+          "13",
+          "14",
+          "15",
+          "16",
+          "16",
+          "17"
+        ],
+        "type": "int64"
+      }
+    },
+    {
+      "name": "constant float32 4D tensor of int8 type",
+      "inputs": {
+        "start": {
+          "data": 0.22992068529129028,
```

Comment:
How about starting from a negative value for this case? (for int8, not uint8 obviously :b)